doc/userguide/src/CreatingTestData/CreatingTestCases.rst

```diff
@@ -215,16 +205,13 @@ example below have arguments `*paths` and `base, *parts`,
 respectively. The former can be used with any number of arguments, but
 the latter requires at least one argument.
 
-.. table:: Keywords with variable number of arguments
-   :class: example
+.. sourcecode:: robotframework
 
-   =============  ================  =================  =================  =================
-     Test Case         Action            Argument           Argument           Argument
-   =============  ================  =================  =================  =================
-   Example        Remove Files      ${TEMPDIR}/f1.txt  ${TEMPDIR}/f2.txt  ${TEMPDIR}/f3.txt
-   \              @{paths} =        Join Paths         ${TEMPDIR}         f1.txt
-   \              ...               f2.txt             f3.txt             f4.txt
-   =============  ================  =================  =================  =================
+   *** Test Cases ***
+   Example
+       Remove Files    ${TEMPDIR}/f1.txt    ${TEMPDIR}/f2.txt    ${TEMPDIR}/f3.txt
+       @{paths} =    Join Paths    ${TEMPDIR}    f1.txt
+       ...           f2.txt    f3.txt    f4.txt
```

Comment:
No need to split arguments to multiple rows (unless the ideas is to demonstrate that feature).